tutorial/practicals/nemo/1_nemo_psyir/README.md

```diff
@@ -240,14 +262,21 @@ are [`Loop`](https://psyclone-ref.readthedocs.io/en/latest/_static/html/classpsy
    assignments = sched.walk(Assignment)
```

Comment:
In line 237: "search for all CodeBlocks in" ?

Reply:
Again in line 257, should it be "all Assignment nodes"?

Reply:
Either is fine.